<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesForm</class>
 <widget class="QDialog" name="PreferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreferencesForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Map theme:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="browsePushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>GeoLiteCity:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="geoLiteCityPathLineEdit"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="mapThemeComboBox">
       <item>
        <property name="text">
         <string>earth/plain/plain.dgml</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>earth/bluemarble/bluemarble.dgml</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>earth/openstreetmap/openstreetmap.dgml</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QDialogButtonBox" name="okCancelButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
